android-app/app/build.gradle

```diff
@@ -35,11 +35,15 @@ android {
 
     buildTypes {
         debug {
+            debuggable true
+            manifestPlaceholders = [usesCleartextTraffic: 'true']
             enableUnitTestCoverage true
             enableAndroidTestCoverage true
             testCoverageEnabled true
         }
         release {
+            debuggable false
```

Comment:
fyi: whenever you've chosen the "release" build type variant, you wont be able to start the debugger anymore. I suggest that you change it locally but keep it `false` whenever we commit anything to github